packages/fx-core/src/question/create.ts

```diff
@@ -232,7 +232,7 @@ function projectTypeQuestion(): SingleSelectQuestion {
         }
       } else {
         staticOptions.push(
-          isOfficeXMLAddinEnabled() && !isOfficeJSONAddinEnabled()
+          (isOfficeXMLAddinEnabled() && !isOfficeJSONAddinEnabled()) || inputs.agent === "office"
```

Comment:
This part is out-of-date with the code in `dev` branch, please rebase both the target branch and the source branch with `dev` branch first because `isOfficeAmlAddinEnabled()` is retired.

Reply:
Thanks for the comments. Just to clarify, do you mean `teams-participant` branch needs to sync the latest change from `dev` branch first, so that I can sync these changes from `teams-participant`.